test/others/test_schema.py

```diff
@@ -395,32 +520,25 @@ def test_table_document_to_dict(table_doc, table_doc_dict):
 
 
 @pytest.mark.unit
-def test_doc_to_json():
+def test_document_from_json(text_doc, text_doc_with_embedding, text_doc_json, text_doc_with_embedding_json):
```

Comment:
I would split this and the following tests in two.
As an example this could become `test_document_from_json_with_embedding(text_doc, text_doc_json)` and `test_document_from_json_without_embedding(text_doc_with_embedding, text_doc_with_embedding_json)`.

Same thing for the ones below.

It's trivial but I'd like this mainly for readability and so that we'll understand right away what is the cause of an eventual failure. 🤞